pyaedt/hfss.py

```diff
@@ -2762,44 +2470,49 @@ def create_perfecth_from_objects(self, startobj, endobject, axisdir=0, sourcenam
+        >>> perfect_h = hfss.create_perfecth_from_objects("perfect1","perfect2",hfss.AxisDir.ZNeg,"Perfect H")
         PyAEDT INFO: Connection Correctly created
         >>> type(perfect_h)
         <class 'pyaedt.modules.Boundary.BoundaryObject'>
 
         """
 
-        if not self.modeler.does_object_exists(startobj) or not self.modeler.does_object_exists(endobject):
+        if not self.modeler.does_object_exists(assignment) or not self.modeler.does_object_exists(reference):
             self.logger.error("One or both objects do not exist. Check and retry.")
             return False
         if self.solution_type in ["Modal", "Terminal", "Transient Network"]:
             sheet_name, point0, point1 = self.modeler._create_sheet_from_object_closest_edge(
-                startobj, endobject, axisdir, bound_on_plane
+                assignment, reference, start_direction, is_boundary_on_plane
             )
 
-            if not sourcename:
-                sourcename = generate_unique_name("PerfH")
-            elif sourcename in self.modeler.get_boundaries_name():
-                sourcename = generate_unique_name(sourcename)
-            return self.create_boundary(self.BoundaryType.PerfectH, sheet_name, sourcename)
+            if not name:
+                name = generate_unique_name("PerfH")
+            elif name in self.modeler.get_boundaries_name():
+                name = generate_unique_name(name)
+            return self.create_boundary(self.BoundaryType.PerfectH, sheet_name, name)
         return None
 
-    @pyaedt_function_handler()
-    def sar_setup(self, Tissue_object_List_ID=-1, TissueMass=1, MaterialDensity=1, voxel_size=1, Average_SAR_method=0):
+    @pyaedt_function_handler(
+        Tissue_object_List_ID="assignment",
+        TissueMass="tissue_mass",
+        MaterialDensity="material_density",
+        Average_SAR_method="average_sar_method",
+    )
+    def sar_setup(self, assignment=-1, tissue_mass=1, material_density=1, voxel_size=1, average_sar_method=0):
         """Define SAR settings.
 
         Parameters
         ----------
-        Tissue_object_List_ID : int, optional
-           The default is ``-1`` to not specify the object.
-        TissueMass : float, optional
-            The default is ``1``.
-        MaterialDensity : optional
-            The default is ``1``.
+        assignment : int, optional
+           Object ID. The default is ``-1`` to not specify the object.
+        tissue_mass : float, optional
+            Mass of tissue in grams. The default is ``1``.
+        material_density : optional
+            Density of material in gram/cm^3. The default is ``1``.
         voxel_size : optional
-            The default is ``1``.
-        Average_SAR_method : optional
+            Size of a voxel in millimeters. The default is ``1``.
+        average_sar_method : optional
+            SAR method. There are two options, ``0`` for IEEE std 1528 and ``1`` for classical Ansys method.
```

Comment:
```suggestion
            SAR method. There are two options, ``0`` for IEEE Standard 1528 and ``1`` for the standard Ansys method.
```
Which is the default?